README.md

```diff
@@ -596,6 +598,33 @@ The DLP templates used in this tutorial are specifically tailored for inspecting
 2. [Create de-identification templates and run de-identification on sample data](https://cloud.google.com/dlp/docs/creating-templates-deid)
 
 
+## Run this pipeline outside Google Cloud shell
+
```

Comment:
We should add a line here stating that google cloud shell come with tools etc., installed If you want to run this solution from your own machine, here are the intstructions.